src/ynEIGEN/TokenStakingNode.sol

```diff
@@ -150,11 +146,6 @@ contract TokenStakingNode is
 
         _fullWithdrawalRoots = tokenStakingNodesManager.delegationManager().queueWithdrawals(_params);
 
-        IYieldNestStrategyManager(tokenStakingNodesManager.yieldNestStrategyManager()).updateTokenStakingNodesBalances(
```

Comment:
Argument why this is no necessary:

As part of queueing withdrawals, no change in total shares happens.

The stakedShares + queuedShares sum is constant.

Therefore the balance `stakedBalance` as computed here remains constant:


```
        StrategyBalance memory _strategyBalance = StrategyBalance({
            stakedBalance: SafeCast.toUint128(_strategiesBalance + _strategiesWithdrawalQueueBalance),
            withdrawnBalance: SafeCast.toUint128(_strategiesWithdrawnBalance)
        });
 ```
 
 Then this computation wouldn't actually update the values specific to this StakingNode for Asset.